context.go

```diff
@@ -445,3 +385,12 @@ func normalizeFlags(flags []Flag, set *flag.FlagSet) error {
 	}
 	return nil
 }
+
+func makeFlagNameVisitor(names *[]string) func(*flag.Flag) {
+	return func(f *flag.Flag) {
+		name := strings.Split(f.Name, ",")[0]
+		if name != "help" && name != "version" {
```

Comment:
Taking these exclusions out breaks no tests, so I'm taking them out until we can find a good reason to have them :smiley_cat: